app.py

```diff
@@ -6,13 +7,13 @@
 import sys
 import traceback
 
+
 app = Flask(__name__)
 
 INTERRUPT_EVENT = Event()
 
-esi_api_host_name = os.environ.get('ESI_API_HOST_NAME') or ""
-esi_api_port = int(os.environ.get('ESI_API_PORT') or "443")
-esi_api_ssl = bool(os.environ.get('ESI_API_SSL') or "true")
+cloud_name = os.environ.get('CLOUD_NAME') or ""
```

Comment:
it might make sense to have the default be something like "openstack"; otherwise the default will never match anything in clouds.yaml, and we probably want to throw an error if `CLOUD_NAME` isn't set

Reply:
Fixed!